### PR TITLE
Fix bug 2074697 - Set-ASSDevice silently terminates when no optional par...

### PR DESCRIPTION
**Pull request by rajsing**
...ams are provided

### CODE REVIEW

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx [hunk 1]
```diff
@@ -513,4 +513,7 @@
   <data name="CIKInvalidWithException" xml:space="preserve">
     <value>Invalid value for Registration Key. CIK provided couldnt be used to decrypt the secrets! The exact exception while decrypting is "{0}"!</value>
   </data>
+  <data name="SetASSDeviceNoSettingsProvided" xml:space="preserve">
```
Comment (avirupch) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: Please replace ASS with AzureStorSimple

Comment (rajsing) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: fixed

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceDetails/SetAzureStorSimpleDevice.cs [hunk 2]
```diff
@@ -135,6 +135,16 @@ public override void ExecuteCmdlet()
...
         private bool ProcessParameters()
         {
+            // Make sure that atleast one of the settings has been provided.
+            // Its no use making a request when the user didnt specify any of
+            // the settings.
+            if (string.IsNullOrEmpty(NewName) && TimeZone == null &&
+                string.IsNullOrEmpty(SecondaryDnsServer) &&
+                (StorSimpleNetworkConfig == null || StorSimpleNetworkConfig.Count() < 1))
+            {
+                throw new ArgumentException(Resources.SetASSDeviceNoSettingsProvided);
```
Comment (parvezah) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceDetails/SetAzureStorSimpleDevice.cs]: Remove ASS from the name used here.

Comment (rajsing) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceDetails/SetAzureStorSimpleDevice.cs]: Had forgotten to add that file in the last checkin, fixed now

**Pull request merged**